_includes/sidebar-data-v2.1.json

```diff
@@ -1571,8 +1571,25 @@
       },
       {
         "title": "Cluster Settings",
-        "urls": [
-          "/${VERSION}/cluster-settings.html"
+        "items": [
+          {
+            "title": "Cluster Settings Overview",
+            "urls": [
+              "/${VERSION}/cluster-settings.html"
+            ]
+          },
+          {
+            "title": "Cost-Based Optimizer",
+            "urls": [
+              "/${VERSION}/cost-based-optimizer.html"
```

Comment:
Curious why CBO is added to cluster settings section?  According to the CBO doc it's on by default in 2.1.